migrations/20240210165254_variants__reference_course_id.sql

```diff
@@ -0,0 +1,10 @@
+ALTER TABLE variants
+ADD COLUMN IF NOT EXISTS course_id bigint;
+
+ALTER TABLE variants
+ADD CONSTRAINT variants_course_id_fkey FOREIGN KEY (course_id) REFERENCES pl_courses (id) ON UPDATE CASCADE ON DELETE CASCADE;
+
+UPDATE variants AS v
+SET course_id = q.course_id
+FROM questions AS q
+WHERE v.question_id = q.id;
```

Comment:
@mwest1066 is there any way to do this that isn't going to completely lock the variants table and thus take down PL? Or is there no way around it and we're just going to have to deploy during a low/no usage time?